src/input/keyboard_and_mouse.rs

```diff
@@ -34,12 +41,9 @@ impl KeyboardAndMouse {
         self.is_cursor_taken
     }
 
-    /// Returns the positions of the mouse clicks during the last interaction.
-    ///
-    /// Clicks performed while the mouse cursor is not available are
-    /// automatically ignored.
-    pub fn left_clicks(&self) -> &[Point] {
-        &self.left_clicks
+    /// Returns true if the cursor is currently within the game window.
+    pub fn is_cursor_within_game_window(&self) -> bool {
```

Comment:
I would drop the `game` part here, I feel it is already understood by the context: `is_cursor_within_window`.